js/pagination.jsx

```diff
@@ -0,0 +1,143 @@
+/*******************************************************************************
+ *
+ * Copyright (c) 2014, Trevor Landau <>
```

Comment:
I have thought about that. 

If we want to make use of `react-paginator` like README says, not only `package.json` but also `browserify.js` should be involved. 

Moztrap is just a UI front-end. IMO, it's too heavy to do that. That's why I modified the source to fit Moztrap. (But I don't know if there is any license concern?)

```javascript
var Paginator = require('react-paginator');

// onPaginate will receive a page number
<Paginator maxPages="5" numPages="10" onClick={this.onPaginate}/>
```